src/HDInsight/HDInsight/ManagementCommands/EnableAzureHDInsightAzureMonitorCommand.cs

```diff
@@ -0,0 +1,140 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.WindowsAzure.Commands.Common;
+using Microsoft.Azure.Commands.HDInsight.Commands;
+using Microsoft.Azure.Management.HDInsight.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.HDInsight.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.HDInsight
+{
+    [Cmdlet("Enable", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightAzureMonitor", DefaultParameterSetName = SetByNameParameterSet, SupportsShouldProcess = true)]
+    [OutputType(typeof(bool))]
+    public class EnableAzureHDInsightAzureMonitorCommand : HDInsightCmdletBase
+    {
+        #region Input Parameter Definitions
+
+        private const string SetByNameParameterSet = "ByNameParameterSet";
```

Comment:
My proposal is to remove all the prefixes "Set" as you cmdlet verb is not "Set".
```
        private const string ByNameParameterSet = "ByNameParameterSet";
        private const string ByResourceIdParameterSet = "ByResourceIdParameterSet";
        private const string ByInputObjectParameterSet = "ByInputObjectParameterSet";
```

Reply:
> My proposal is to remove all the prefixes "Set" as you cmdlet verb is not "Set".
> 
> ```
>         private const string ByNameParameterSet = "ByNameParameterSet";
>         private const string ByResourceIdParameterSet = "ByResourceIdParameterSet";
>         private const string ByInputObjectParameterSet = "ByInputObjectParameterSet";
> ```

Fixed. Thanks